core-services/prow/02_config/_plugins.yaml

```diff
@@ -190,7 +190,7 @@ approve:
   - openshift-priv/openshift-tests-private
   - openshift/openshift-tests
   - openshift/openshift-tests-private
-  require_self_approval: true
+  require_self_approval: false
```

Comment:
This is unrelated to the the title of the PR and has no justification. Do you know what this does?

Also, I don't think you should be changing approval settings for `openshift/openshift-tests`.

Reply:
Yes, I know. I hope the OWNERS don't need to approve their PR explicitly. In other words, their PR can be approved automatically.
> Also, I don't think you should be changing approval settings for openshift/openshift-tests.

Why? As you know, only QE is using this repo now.

Reply:
Anyway, I removed this setting change.